samples/GuestConfiguration/audit-installed-application-linux/README.md

```diff
@@ -0,0 +1,42 @@
+# Audit for installed application - Linux
+
+This policy audits the results of a check for applications installed in Linux virtual machines.
+
+This policy requires another definition to also be deployed that loads requirements for performing audits inside virtual machines.  Please see the policy sample
+[Deploy extension to audit for installed applications - Linux](..\deployextension-installed-application-linx\README.md).
```

Comment:
Does this link work?  I believe it needs to be ../ instead of ..\

Reply:
"linx" should be "linux" as well

Reply:
fixed